src/lib/util/StaticResourceHandler.ts

```diff
@@ -40,10 +40,10 @@ export class StaticResourceHandler {
 	}
 
 	public identifyBufferType(buffer: Buffer): StaticResourceType {
-		if (isText(null, buffer)) {
-			return StaticResourceType.TEXT;
-		} else if (isZip(buffer)) {
+		if (isZip(buffer)) {
```

Comment:
@jfeingold35 This switches the logic because the new library doesn't have an `isText`, but I think it's equivalent.

Reply:
The test cases for this class are pretty comprehensive. So if those are all passing, then it's probably an acceptable substitution. Good find!